widget-mvc.el

```diff
@@ -147,7 +149,7 @@ CTX is a `wmvc:context'. If nil, use `current-language-environment'."
     (cdr (assq name attrs))))
 
 (defun wmvc:context-widget-map-add (context name widget)
```

Comment:
@kiwanami 
I will update prefix name to standard naming conventions: `widget-mvc`, tell me if you oppose this.